# Solution to 02_jupyter.md

My research question is "How long does it take my laptop to sum a set of numbers?"

In [2]:
data = [1,2,3,4,5]

In [3]:
import numpy as np
%timeit np.sum(data)

2.54 µs ± 44 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)


I inserted this cell using only keyboard shortcuts after the cell below was complete. Yay!

It took just over 2.5 microseconds on May 27, 2026.